lib/login.ts

```diff
@@ -951,7 +951,7 @@ export function loginWithAppServiceMSI(options?: MSIAppServiceOptions | Callback
  */
 export async function execAz(cmd: string): Promise<any> {
   return new Promise<any>((resolve, reject) => {
-    exec(`az ${cmd} --out json`, { encoding: "utf8" }, (error, stdout) => {
+    execFile(`az`, [`${cmd}`, `--out json`], { encoding: "utf8" }, (error, stdout) => {
```

Comment:
Now that `cmd` is going into an array of strings, we don't really need the template literals do we?

```suggestion
    execFile(`az`, [cmd, `--out json`], { encoding: "utf8" }, (error, stdout) => {
```

Reply:
@ramya-rao-a Yes, I think so :-)